cookbooks/swift/recipes/default.rb

```diff
@@ -17,3 +17,7 @@
   user node['username']
   group node["username"]
 end
+
+if node["tracing"] then
+  include_recipe "swift::tracing_info"
+end
```

Comment:
Yeah, its just some useful info for the user. Wanted to put info somewhere. maybe I should just add it to the readme. But thought this might be useful. Happy to remove it if it's too confusing.

Reply:
It's still in the same place, but now also starts the Jaeger all-in-one inside the vm.